clifford/__init__.py

```diff
@@ -1067,8 +1035,7 @@ def __mul__(self, other):
             newValue = self.layout.gmt_func(self.value,other.value)
         else:
             if issubclass(other.__class__, np.ndarray):
-                obj = np.empty(1, dtype=object)
-                obj[0] = self
+                obj = self.__array__()
```

Comment:
`obj = np.asanyarray(self)` might be more idiomatic - usually you shouldn't invoke `__dunder__` methods directly